resolwe_bio/processes/variant_calling/variants_to_table.py

```diff
@@ -0,0 +1,123 @@
+"""Run GATK VariantsToTable."""
+
+from plumbum import TEE
+
+from resolwe.process import (
+    BooleanField,
+    Cmd,
+    DataField,
+    FileField,
+    GroupField,
+    ListField,
+    Process,
+    SchedulingClass,
+    StringField,
+)
+
+
+class GatkVariantsToTable(Process):
+    """Run GATK VariantsToTable.
+
+    This tool extracts specified fields for each variant in a VCF file
+    to a tab-delimited table, which may be easier to work with than a VCF.
+    For additional information, please see
+    https://gatk.broadinstitute.org/hc/en-us/articles/360036711531-VariantsToTable
+    """
+
+    slug = "variants-to-table"
+    name = "GATK VariantsToTable"
+    category = "GATK"
+    process_type = "data:variants:vcf:variantstotable"
```

Comment:
The output of the process does not include variants in the vcf file. It would be more appropriate to use a new data type. Something like `data:variantstable` or `variantstotable`.